2018-04-24 14:57:33.860 - pyansibleinv -        dbbackup_mysql_list -  INFO - args:{'--cluster_id': None,
 '--data_host': 'dcs-db-c00aee9f.iad1:10.41.234.57',
 '--dbfqdn': 'dcs-db-c00aee9f.iad1',
 '--dbtype': 'mysql',
 '--ssh_try_limit': '600',
 '--sshkey': '/opt/ansible/db.pem',
 '--sshpass': 'LR1!Ika',
 '--taskid': None,
 '--template_only': False,
 '--workdir': '/opt/ansible',
 'dbbackup': True,
 'debug': False,
 'list': True}
2018-04-24 14:57:33.861 - pyansibleinv -        dbbackup_mysql_list -  INFO - dict:{'ssh_try_limit': '600', 'workdir': '/opt/ansible', 'sshkey': '/opt/ansible/db.pem', 'template_only': False, 'dbtype': 'mysql', 'backup': True, 'taskid': None, 'task_id': '  external_task_id: 2815fb8f-cd36-4b5a-8dad-af2f1ef57437\n', 'st': True, 'cluster_id': None, 'sshpass': 'LR1!Ika', 'dbfqdn': 'dcs-db-c00aee9f.iad1', 'data_host': 'dcs-db-c00aee9f.iad1:10.41.234.57', 'bug': False, 'uuid': '2815fb8f-cd36-4b5a-8dad-af2f1ef57437'}
2018-04-24 14:57:33.861 - pyansibleinv -        dbbackup_mysql_list -  INFO - create ansible hosts: /opt/ansible/inventory/2815fb8f-cd36-4b5a-8dad-af2f1ef57437/hosts
2018-04-24 14:57:33.861 - pyansibleinv -        dbbackup_mysql_list -  INFO - craete ansible playbooks: /opt/ansible/dbbackup_mysql_list_2815fb8f-cd36-4b5a-8dad-af2f1ef57437.yml
2018-04-24 14:57:33.863 - pyansibleinv -        dbbackup_mysql_list -  INFO - check ssh availability
2018-04-24 14:57:33.863 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - Attempting to connect to 10.41.234.57 on port 22
2018-04-24 14:57:33.931 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - Connected to 10.41.234.57 on port 22
2018-04-24 14:57:33.932 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - Attempting to connect to 10.41.234.57 on port 22
2018-04-24 14:57:33.998 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - Connected to 10.41.234.57 on port 22
2018-04-24 14:57:33.998 - pyansibleinv -        dbbackup_mysql_list -  INFO - run ansible from python
2018-04-24 14:57:34.605 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - 
2018-04-24 14:57:34.605 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - PLAY [list backup sets] ***********************************************************************************************************************************************************************
2018-04-24 14:57:35.366 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - 
2018-04-24 14:57:35.366 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - TASK [list mysql backup set] ******************************************************************************************************************************************************************
2018-04-24 14:57:38.203 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - [0;33mchanged: [dcs-db-c00aee9f.iad1][0m
2018-04-24 14:57:38.916 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - 
2018-04-24 14:57:38.916 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - TASK [Record backup result] *******************************************************************************************************************************************************************
2018-04-24 14:57:39.106 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - [0;32mok: [dcs-db-c00aee9f.iad1][0m
2018-04-24 14:57:39.129 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - 
2018-04-24 14:57:39.129 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - PLAY RECAP ************************************************************************************************************************************************************************************
2018-04-24 14:57:39.130 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - [0;33mdcs-db-c00aee9f.iad1[0m       : [0;32mok=2   [0m [0;33mchanged=1   [0m unreachable=0    failed=0
2018-04-24 14:57:39.228 - pyansibleinv - dbbackup_mysql_list_stdout -  INFO - --- Total Excution time: 0:00:05.368188 ---
2018-04-24 14:57:39.267 - pyansibleinv - dbbackup_mysql_list_stderr - ERROR - {'dcs-db-c00aee9f.iad1': {u'Playbook ID': u'2815fb8f-cd36-4b5a-8dad-af2f1ef57437', u'Value': {u'backup_set02': {u'2018_04_23-07_45_55': {u'incrementals': [{u'2018_04_23-09_41_33': {u'size': u'2.9 MB'}}, {u'2018_04_23-09_41_19': {u'size': u'4.6 MB'}}], u'size': u'2.1 GB'}, u'total_szie': u'2.1 GB'}, u'backup_set01': {u'2018_04_23-09_42_05': {u'incrementals': [], u'size': u'2.1 GB'}, u'total_szie': u'2.1 GB'}}, u'Playbook Path': u'/opt/ansible/dbbackup_mysql_list_2815fb8f-cd36-4b5a-8dad-af2f1ef57437.yml', u'Key': u'mybak_dcs-db-c00aee9f.iad1', u'Type': u'text', u'ID': u'fe0dc700-9521-4206-a4a3-a2760bb8f07e'}}
